api/src/main/java/org/geysermc/geyser/api/util/CreativeCategory.java

```diff
@@ -26,32 +26,35 @@
 package org.geysermc.geyser.api.util;
 
 import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
  * Represents the creative menu categories or tabs.
  */
 public enum CreativeCategory {
+    ALL("all", 0),
     CONSTRUCTION("construction", 1),
     NATURE("nature", 2),
     EQUIPMENT("equipment", 3),
     ITEMS("items", 4),
+    ITEM_COMMAND_ONLY("item_command_only", 5),
```

Comment:
I wonder how this one actually works - will ask around. If it's just a way to hide the item in the creative category, we should document it